README.md

```diff
@@ -10,10 +10,34 @@ The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high
 
 Deploy the charm:
 ```bash
-juju deploy vault-k8s -n 5 --trust
+juju deploy vault-k8s --channel edge -n 5 --trust
 ```
 > Note: It is advised to deploy Vault with an odd number of units
 
+### Access Vault through Ingress
+
+Deploy Traefik
+
+```bash
+juju deploy traefik-k8s --channel edge --trust --config external_hostname=<your hostname> 
+```
+
+Integrate Traefik with Self-Signed-Certificates Operator
+
+```bash
+juju deploy self-signed-certificates --channel beta
+juju integrate self-signed-certificates:certificates traefik-k8s:certificates
+```
+
+Integrate Vault with Traefik
+
+```bash
+juju integrate vault-k8s:send-ca-cert traefik-k8s:receive-ca-cert
+juju integrate vault-k8s:ingress traefik-k8s:ingress
+```
+
+You should now be able to access the Vault at `https://<your hostname>/<model name>-vault-k8s.`
```

Comment:
I wouldn't hardcode this, I'd suggest instead leveraging Traefik `show-proxied-endpoint` Juju action to get the Vault address.

Reply:
OK

Reply:
Done